### PR TITLE
fix staticcheck in   test/e2e/cloud/

### PR DESCRIPTION
**Pull request by tanjunchen**
**What type of PR is this?**
 
/kind cleanup

**What this PR does / why we need it**:

Fix static check failures  in test/e2e/cloud/

**Which issue(s) this PR fixes**:

Ref: #81657

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: test/e2e/cloud/gcp/resize_nodes.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ var _ = SIGDescribe("Nodes [Disruptive]", func() {
 		systemPods, err := e2epod.GetPodsInNamespace(c, ns, map[string]string{})
 		framework.ExpectNoError(err)
 		systemPodsNo = int32(len(systemPods))
-		if strings.Index(framework.TestContext.CloudConfig.NodeInstanceGroup, ",") >= 0 {
```
Comment (tanjunchen) [inline test/e2e/cloud/gcp/resize_nodes.go]: staticcheck show that we neet to change this and   test/cloud not in  hack/.staticchek_failures

**Comment by tanjunchen** [comment]
@neolit123

**Review by SataQiu — COMMENTED** [review]
Thanks @tanjunchen 
/lgtm

**Comment by liggitt** [comment]
please remove the package from the staticcheck failures file

**Comment by tanjunchen** [comment]
@liggitt 
test/cloud not in hack/.staticchek_failures,i use staticcheck  not  hack/.. command.

**Comment by liggitt** [comment]
> @liggitt
> test/cloud not in hack/.staticchek_failures,i use staticcheck not hack/.. command.

oh, I see. in the future, I would recommend focusing on the issues identified by the hack static check command

/lgtm
/approve

**Comment by tanjunchen** [comment]
@liggitt 
get it.Thanks!

**Pull request merged**